.npmrc-cloud

```diff
@@ -0,0 +1,5 @@
+auto-install-peers=true
+strict-peer-dependencies=false
+@taskforcesh:registry=https://npm.taskforce.sh/
```

Comment:
What's this file for?

Reply:
ping @scopsy

Reply:
we define here the npm registry from where the packages will be loaded, especially `@taskforcesh/bullmq-pro`